### DISCUSSION TITLE
关于鼠标手势存储的数据格式的疑问

### DISCUSSION THREAD

**Discussion by LCC456** [shuax/MouseInc.Settings#105] [Q&A]
鼠标手势是以数组的格式保存的，请问数组中每个数据各表示什么，作者大大能告知下学习资料吗？感谢

**Comment by shuax** [comment]
依次是x,y,x,y坐标

**Reply by LCC456** [reply]
懂了，谢谢！一开始以为用的是数学的直角坐标系，发现上下是相反的。真正的坐标系是原点向右为x轴的递增，原点向下为y轴的递增。
